hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HStore.java

```diff
@@ -923,12 +921,12 @@ private void bulkLoadHFile(StoreFile sf) throws IOException {
   }
 
   @Override
-  public ImmutableCollection<StoreFile> close() throws IOException {
```

Comment:
The Store interface specifies a return type of Collection. Guava's ImmutableCollection extends Collection, sure, but unnecessarily binds all (private) users of HStore#close to Guava, and exposes this derived interface to the vagaries of Guava's evolution over time. Fix.